src/envo/envo-edit.owl

```diff
@@ -12617,53 +12592,51 @@ AnnotationAssertion(owl:deprecated <http://purl.obolibrary.org/obo/ENVO_00000886
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000887> (large river biome)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000887> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000887> "ENVO:00000887"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000887> "A freshwater river biome which is determined by a large river."@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_00000887> "wwfBiome")
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_00000887> "\"Large\" is ambiguous. For details on \"Large rivers\" (e.g. the Mekong river) see http://worldwildlife.org/biomes/large-river-ecosystems This class will be replaced with a less ambiguous class."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000887> "large river biome"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000887> <http://purl.obolibrary.org/obo/ENVO_01000253>)
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000888> (large river headwater biome)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000888> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000888> "ENVO:00000888"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000888> "A freshwater river biome which is determined by the headwaters of a large river."@en)
```

Comment:
since the headwaters of a large river a likely to be small rivers, how is this different from a 'small river biome'?